image/store.go

```diff
@@ -170,6 +171,9 @@ func (is *store) Search(term string) (ID, error) {
 
 	dgst, err := is.digestSet.Lookup(term)
 	if err != nil {
+		if strings.Contains(err.Error(), "digest not found") {
```

Comment:
can we make `digestSet.Lookup` to return a typed error to avoid this string comparison?